python/az/aro/azext_aro/custom.py

```diff
@@ -134,9 +137,43 @@ def aro_create(cmd,  # pylint: disable=too-many-locals
                        parameters=oc)
 
 
-def aro_delete(client, resource_group_name, resource_name, no_wait=False):
+def aro_delete(cmd, client, resource_group_name, resource_name, no_wait=False):
```

Comment:
`cmd` as a first parameter is apparently magic :crystal_ball: https://github.com/Azure/azure-cli/blob/dev/doc/authoring_command_modules/authoring_commands.md#write-a-command